recipes-soletta/soletta/soletta_git.bb

```diff
@@ -38,17 +42,121 @@ FILES_${PN}-dbg = " \
 "
 
 FILES_${PN}-dev = " \
-                ${datadir}/soletta/* \
+                ${datadir}/soletta/web-inspector/* \
```

Comment:
If you put ${PN}-flow-gtk before ${PN}-dev in PACKAGES, then this entire change here will not be necessary: files get put into the first package that matches their file name, so the latter packages (like -dev) can use fairly generic wildcards.